snap/snapcraft.yaml

```diff
@@ -1,18 +1,13 @@
 name: anbox
-version: 4
-version-script: |
-  if [ "$SNAPCRAFT_GRADE" = "stable" ]; then
-    echo $SNAPCRAFT_PROJECT_VERSION
-  else
-    echo $SNAPCRAFT_PROJECT_VERSION-$(git rev-parse --short HEAD)
-  fi
+version: git
```

Comment:
Why are we dropping the actual version here? As there are no stable releases of Anbox we should fix what we had so far an use `4+gitr<git revision>`